openqa-label-known-issues

```diff
@@ -121,7 +121,7 @@ handle_unknown() {
     echoerr "$i : Unknown issue, to be reviewed -> $i/file/autoinst-log.txt"
     echoerr -e "Likely the error is within this log excerpt, last lines before shutdown:\n---"
     # Look for different termination points with likely context
-    (grep -A 12 'Backend process died, backend errors are reported below in the following lines' "$out" || grep -B 10 'sending magic and exit' "$out" || grep -B 5 'killing command server.*because test execution ended through exception' "$out" || grep -B 5 'EXIT 1' "$out" || grep -B 10 '\(Result: died\|isotovideo failed\)' "$out") | head -n -1 >&2
+    (grep -A 12 'Backend process died, backend errors are reported below in the following lines' "$out" || grep -B 10 'sending magic and exit' "$out" || grep -B 5 'killing command server.*because test execution ended through exception' "$out" || grep -B 5 'EXIT 1' "$out" || grep -B 10 '\(Result: died\|isotovideo failed\)' "$out" || echo '') | head -n -1 >&2
```

Comment:
Better call `echoerr '(No log excerpt found)'`

Reply:
`echoerr` doesn't make sense because the output would not be considered by head and the `>&2` causes everything to be written to `stderr` anyways.

Reply:
you are right, resolved